community/kernel/src/test/java/org/neo4j/kernel/builtinprocs/BuiltInProceduresTest.java

```diff
@@ -82,7 +81,18 @@ public void shouldListAllIndexes() throws Throwable
 
         // When/Then
         assertThat( call("db.indexes"),
-            contains( record( "INDEX ON :User(name)", "ONLINE" ) ) );
+            contains( record( "INDEX ON :User(name)", "ONLINE", "node_label_property" ) ) );
+    }
+
+    @Test
+    public void shouldListAllUniqueIndexes() throws Throwable
+    {
+        // Given
+        givenUniqueConstraint( "User", "name" );
+
+        // When/Then
+        assertThat( call( "db.indexes" ),
+                contains( record( "INDEX ON :User(name)", "ONLINE", "node_unique_property" ) ) );
```

Comment:
Why don't you use the enum constants in this assertion? The test will fail if the enum is renamed.